src/workerd/api/url-standard.h

```diff
@@ -95,7 +95,19 @@ class URLSearchParams: public jsg::Object {
     JSG_METHOD(toString);
     JSG_ITERABLE(entries);
 
-    if (flags.getUrlSearchParamsDeleteHasValueArg()) {
+    if (!flags.getSpecCompliantUrl()) {
+      // This is a hack. The spec-compliant URLSearchParams type is used in
+      // the Body constructor, see:
+      // https://github.com/balusch/workerd/tree/main/src/workerd/api/http.h#L255
```

Comment:
Maybe we could specify a specific commit of the official repository instead of using the `main` branch, so that the reference remains stable and is not affected by subsequent commits.